### DISCUSSION TITLE
Ouverture fichier PES avec Inkstitch

### DISCUSSION THREAD

**Discussion by GDAYDE** [inkstitch/inkstitch#937] [Q&A]
Bonjour,
Je viens d'installer Inkscape ainsi que le module Inkstitch, mais je n'arrive pas à ouvrir les fichiers PES, j'ai ce message d'erreur 

 Traceback (most recent call last):
  File "D:\a\inkstitch\inkstitch\inkstitch.py", line 44, in <module>
  File "D:\a\inkstitch\inkstitch\lib\extensions\input.py", line 50, in affect
  File "C:\hostedtoolcache\windows\Python\2.7.18\x86\lib\encodings\utf_8.py", line 16, in decode
UnicodeDecodeError: 'utf8' codec can't decode byte 0xc9 in position 23: invalid continuation byte

Je ne sais pas trop ce que je dois faire, quelqu'un peut-il m'aider ? Merci par avance

**Comment by tyrosinase** [comment]
On 12/25/20 10:54 AM, GDAYDE wrote:
>
> Bonjour,
> Je viens d'installer Inkscape ainsi que le module Inkstitch, mais je 
> n'arrive pas à ouvrir les fichiers PES, j'ai ce message d'erreur
>
> Traceback (most recent call last):
> File "D:\a\inkstitch\inkstitch\inkstitch.py", line 44, in
> File "D:\a\inkstitch\inkstitch\lib\extensions\input.py", line 50, in 
> affect
> File 
> "C:\hostedtoolcache\windows\Python\2.7.18\x86\lib\encodings\utf_8.py", 
> line 16, in decode
> UnicodeDecodeError: 'utf8' codec can't decode byte 0xc9 in position 
> 23: invalid continuation byte
>
> Je ne sais pas trop ce que je dois faire, quelqu'un peut-il m'aider ? 
> Merci par avance
>

Ink/Stitch has a UTF-8 bug, and can't currently handle extended 
characters in file names or paths. The workaround is to rename the file 
to remove any "à" and so forth, and/or to move it to a directory that 
doesn't include extended characters.

Ink / Stitch a un bogue UTF-8 et ne peut actuellement pas gérer les 
caractères étendus dans les noms de fichiers ou les chemins. La solution 
de contournement consiste à renommer le fichier pour supprimer tout "à" 
et ainsi de suite, et / ou à le déplacer vers un répertoire qui n'inclut 
pas de caractères étendus. (GTranslate)

**Comment by GDAYDE** [comment]
merci pour votre réponse, malheureusement  là je ne trouve pas comment
faire le contournement, je continue à chercher

*Ghislaine Daydé*
06 70 57 88 68


Le ven. 25 déc. 2020 à 20:27, Tyrosinase <notifications@github.com> a
écrit :

**Comment by kaalleen** [comment]
I know you are a beginner and maybe it's not recommended to use Ink/Stitch versions which are still in development.
But the windows version should work in the current state already. If you don't mind to give it a try, download and install this version of Ink/Stitch here:

https://github.com/inkstitch/inkstitch/releases/download/dev-build-kaalleen-inkscape1.0/inkstitch-kaalleen-inkscape1.0-windows-fr_FR.zip

I would be very curious if it solves your issue.

**Comment by GDAYDE** [comment]
Voilà, j'ai trouvé la solution à mon problème, dans le chemin de mes
fichiers PES il y avait un accent, j'ai déplacé les fichiers dans un
répertoire sans accent, sur mon disque dur et ça marche ! J'ai pu ouvrir
mes fichiers broderie avec Ink/stitch et ajouter des éléments.
Il me reste maintenant à approfondir le logiciel mais cela ne me semble pas
très difficile, en respectant bien les étapes
Merci pour votre réponse

*Ghislaine Daydé*
06 70 57 88 68
